System/Serializable.h

```diff
@@ -47,18 +47,16 @@ namespace RTE {
 				reader.ReportError("Wrong type in Reader when passed to Serializable::Create()");
 				return -1;
 			}
-
 			// This is the engine for processing all properties of this Serializable upon read creation.
 			while (reader.NextProperty()) {
+				m_CurrentlyReadFileAndLine = ("in file " + reader.GetCurrentFilePath() + " on line " + std::to_string(reader.GetCurrentFileLine()));
```

Comment:
I feel like the naming of m_CurrentlyReadFileAndLine is kinda bad, in that it has all sorts of extra stuff around it. I don't mind building the descriptive string here, but maybe try to find a name that reflects that it's not just the file name and line number.

Reply:
I wanted to call it `m_ReaderPosition` but thought you will be angery.
Appears you are angery regardless. :shrug:

Reply:
Hahaha yep
I'm not exactly sure what to rename it to, let's chat in discord and maybe figure something out.

Reply:
Resulting decision was m_FormattedReaderPosition.